frontend/explorer/public/js/main.js

```diff
@@ -119,6 +120,139 @@ function appendUnlabeledStat(table, text) {
 	return [tr, textCell];
 }
 
+
+// appendNavigationButtons appends two buttons that navigate between the previous and the next block.
```

Comment:
please remove commented out code, either use it or delete it

Reply:
Deleted.

Reply:
Deleted.